shacl12-sparql/index.html

```diff
@@ -1315,6 +1315,81 @@ <h3>Select Expressions</h3>
 					</div>
 				</aside>
 			</section>
+
+			<section id="EvalExpression">
+				<h3>Eval Expressions</h3>
+				<p>
+					A <a>node expression</a> that has a <a>value</a> for <code>sh:eval</code> is called an <dfn>eval expression</dfn> with the <a>function name</a>
+					<code>sh:EvalExpression</code>.
+				</p>
+				<p class="syntax">
+					<span data-syntax-rule="EvalExpression-syntax-eval">A node in an RDF graph
+					is a <a>well-formed</a> <a>eval expression</a>
+					if it is a <a>blank node</a>
+					that is the <a>subject</a> of exactly one <a>triple</a>
+					with <code>sh:eval</code> as <a>predicate</a>
+					and a <a>literal</a> with <a>datatype</a> <code>xsd:string</code> as <a>object</a>.</span>
```

Comment:
I still can't tell whether this parenthetical (or its negative) is intended to be understood from the curren text. Therefore, I think this parenthetical (or its negative) should be added.
```suggestion
					and a <a>literal</a> with <a>datatype</a> <code>xsd:string</code> as <a>object</a>.
					(It may also be the <a>subject</a> of multiple <a>triples</a> 
					with other <a>predicate</a> and/or <a>object</a> values.)</span>
```

Reply:
This would be adding the negative.
```suggestion
					and a <a>literal</a> with <a>datatype</a> <code>xsd:string</code> as <a>object</a>.
					(It may not be the <a>subject</a> of any <a>triples</a> 
					with other <a>predicate</a> and/or <a>object</a> values.)</span>
```

Reply:
Will change this to "... a blank node that has exactly one value for the predicate sh:eval and that value is a literal with datatype xsd:string". I hope this clarifies it enough.

In general, if something is not stated then it is allowed. Nothing prevents such blank nodes to have other properties, e.g. an rdf:type or annotations, and that is by design.